go.sum

```diff
@@ -576,6 +576,8 @@ github.com/vesoft-inc/nebula-go/v3 v3.0.0-20220420060617-5427e0756333 h1:v9VMLnW
 github.com/vesoft-inc/nebula-go/v3 v3.0.0-20220420060617-5427e0756333/go.mod h1:+sXv05jYQBARdTbTcIEsWVXCnF/6ttOlDK35xQ6m54s=
 github.com/vesoft-inc/nebula-go/v3 v3.0.0 h1:ii5T3vps4xAQZkzPvGn6NuiUWlH/rm1zdIS5VTEA71A=
 github.com/vesoft-inc/nebula-go/v3 v3.0.0/go.mod h1:+sXv05jYQBARdTbTcIEsWVXCnF/6ttOlDK35xQ6m54s=
```

Comment:
execute go mod tidy to clear unnecessary sum?

Reply:
Got it